src/cordova/test.action.mjs

```diff
@@ -51,7 +51,7 @@ export async function main(...parameters) {
     'clean',
     'test',
     '-scheme',
-    APPLE_LIBRARY_NAME,
+    `${APPLE_LIBRARY_NAME}-Package`,
```

Comment:
This had to change because now the package has two products, one with the same name as the package, so the auto-generated scheme uses -Package for the package scheme. The library doesn't have the tests.